samples/suit/ab/README.rst

```diff
@@ -169,332 +379,263 @@ Building and programming using the command line
 
 To build and program the sample to the nRF54H20 DK, complete the following steps:
 
-.. tabs::
+1. |open_terminal_window_with_environment|
```

Comment:
What I am missing is a description how to update the nordic top manifest.
There is also no mention in the document, that it is not possible to use the `nordic_top.suit` manifest directly to update the nordic elements (configurable by `CONFIG_SUIT_NORDIC_TOP_INDEPENDENT_UPDATE_FORBIDDEN`, which is *not* a sample specific option.

Reply:
As mentioned before, there will be a follow-up with the extended `testing` section, including raw pushes and Nordic top updates.